charts/substra-backend/values.yaml

```diff
@@ -856,12 +856,19 @@ docker-registry:
 minio:
   enabled: false
   auth:
-    rootUser: minio
-    rootPassword: minio1234
+    rootUser: &objecStoreRootUser minio
+    rootPassword: &objectStoreRootPassword minio1234
```

Comment:
How does that works? it concatenates the content of the env var + `minio`, or is `minio` the default value if the env var is not set?

Reply:
It creates two re-usable variables inside of the `values.yaml` file `objecStoreRootUser ` and `objectStoreRootPassword ` with values `minio ` and `minio1234`. I was creates those variables to test if the CI issue is related to the missing environment variables and it seems like it, I'll cleanup those changes to make it re-usable for chart users as well :)

Reply:
Thanks for the explanation, very clear 👍